pkgs/development/python-modules/python-magic/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage, lib, fetchPypi, file, stdenv }:
+
+buildPythonPackage rec {
+  pname = "python-magic";
+  version = "0.4.13";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "128j9y30zih6cyjyjnxhghnvpjm8vw40a1q7pgmrp035yvkaqkk0";
+  };
+
+  propagatedBuildInputs = [ file ];
```

Comment:
I don't think this is necessary. Is it, @FRidh?

Reply:
No, it is not. The patch introduces it so it can be found during run-time. I doubt it needs to be explicitly added during build-time.